modules/exploits/linux/http/magnusbilling_unauth_rce_cve_2023_30258.rb

```diff
@@ -0,0 +1,191 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'rex/stopwatch'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Format::PhpPayloadPng
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Magnusbilling application unauthenticated Remote Command Execution.',
+        'Description' => %q{
+          A Command Injection vulnerability in magnusbilling application 6.x and 7.x allows
```

Comment:
```suggestion
          A Command Injection vulnerability in MagnusBilling application 6.x and 7.x allows
```

Reply:
Please, would you mind capitalizing all the instances of `magnusbilling` in the documentation and the module too?

Reply:
See https://github.com/rapid7/metasploit-framework/pull/18481/commits/bfff35eb63c478194e5a76b702be7dde2dd5728d.